changelogs/api-management-service.md

```diff
@@ -1,5 +1,51 @@
 # Azure API Management service changelog
 
+## Release - API Management service: July, 2022
+
+A regular Azure API Management service update was started on July 20, 2022. It may take several weeks for your API Management service to receive the update.
+
+### New features, fixes, and improvements
+
+1. We optimized the loading time of API schemas for management plane (including Azure portal) and developer portal operations.
+2. We increased the maximum length of each URL path segment from 520 to 1024 characters.
+3. We fixed an issue, where API Management allowed creation of multiple API versions with empty identifiers within one API version set.
+4. We fixed an issue, where API Management deserialized C-style hex strings in exported OpenAPI files as hex values.
+5. We fixed an issue, where API Management failed to export OpenAPI definitions if referenced schemas didn't have the `typename` property defined.
```

Comment:
For VNET enabled customers, we are now monitoring Inbound connectivity using Network Status

![image](https://user-images.githubusercontent.com/6847558/181181032-c3c9f471-2c64-4dee-9ba3-41196f202aaf.png)